bugsnag.go

```diff
@@ -44,17 +44,22 @@ const skipStackFrames = 4
 
 // Fire forwards an error to Bugsnag. Given a logrus.Entry, it extracts the
 // "error" field (or the Message if the error isn't present) and sends it off.
+// It also send all the logger data into an additional tab.
 func (hook *bugsnagHook) Fire(entry *logrus.Entry) error {
 	var notifyErr error
-	err, ok := entry.Data["error"].(error)
-	if ok {
+	if errString, ok := entry.Data["error"].(string); ok {
+		notifyErr = errors.New(errString)
+	} else if err, ok := entry.Data["error"].(error); ok {
```

Comment:
We send data to bugsnag based on log entry. So we may add a field error and write it as a string.
In fact, we are doing it many times.
I think it's best to send errors on the error field, but in case of string, it's to sad to not get any value.
What do you think?